File: ACCENTURE CODING/cleanString.cpp
#include <iostream>
#include <string>
#include <unordered_set>
using namespace std;

string cleanString(const string& A, const string& B) {
    unordered_set<char> charactersB(B.begin(), B.end());
    string result;

    for (char ch : A) {
        if (charactersB.find(ch) == charactersB.end()) {
            result += ch;
        }
    }

    return result.empty() ? "Empty" : result;
}

int main() {
    string A = "HELLO";
    string B = "LO";
    
    string leftover = cleanString(A, B);
    cout << leftover << endl; // Output: "HE"

    return 0;
}

File: ACCENTURE CODING/cleanString.py
def clean_string(A, B):
    result = ''
    
    # Check each character in A
    for ch in A:
        # If the character is not in B, add it to the result
        if ch not in B:
            result += ch
    
    # If result is empty, return "Empty"
    if not result:
        return "Empty"
    
    return result

# Example usage:
A = "HELLO"
B = "LO"
print(clean_string(A, B))  # Output: "HE"
